.github/workflows/scalar-functional-tests.yml

```diff
@@ -20,7 +20,7 @@ jobs:
       fail-fast: false
       matrix:
         # Order by runtime (in descending order)
-        os: [windows-2019, macos-10.15, ubuntu-16.04, ubuntu-18.04, ubuntu-20.04]
+        os: [windows-2019]
```

Comment:
We should restrict this even further. In my tests, running _just_ `--test=GitCommandsTests(SparseMode).AddFileAndCommitOnNewBranchSwitchDeleteFolderAndSwitchBack()` already triggered the error, and this could potentially speed up our edit/push/analyze cycle.

Reply:
This is a consistent failure? Ok.

1. Create a new branch in `microsoft/scalar` (or your fork of it)
2. Add `[Category("BROKEN")]` to this test.
3. In `Scalar.FunctionalTests/Program.cs` change this line:

```
            HashSet<string> includeCategories = new HashSet<string>() { "BROKEN" };
```

4. Point your `microsoft/git` branch to clone that Scalar branch for the functional tests.

Reply:
Nice!